tutorials/synthetic-retrieval-evaluation-customization/notebooks/synthetic_data_generation_retriever_evaluation.ipynb

```diff
@@ -0,0 +1,487 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/vinay/.local/lib/python3.10/site-packages/dask/dataframe/__init__.py:42: FutureWarning: \n",
+      "Dask dataframe query planning is disabled because dask-expr is not installed.\n",
+      "\n",
+      "You can install it with `pip install dask[dataframe]` or `conda install dask`.\n",
+      "This will raise in a future version.\n",
+      "\n",
+      "  warnings.warn(msg, FutureWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
```

Comment:
The notebook should have text in between the code cells describing what is happening at each stage to walk the user through it. Reference the [image curation](https://github.com/vinay-raman/NeMo-Curator/blob/retriever_evalset_generations/answerability_filter/tutorials/image-curation/image-curation.ipynb) tutorial as an example.